active/community.md

```diff
@@ -0,0 +1,75 @@
+# WG Charter Template
+
+## Scope of responsibilities
+
+This group exists to support the official community platforms and those in elevated roles on those platforms.
+
+The Forum and the Discord are core parts of the overall community and those that help and moderate these platforms take significant time from their day to help others. We can support them through training and taking action to make their lives easier.
+
+The broad goals of the group are:
+  - facilitate and centralise discussion between moderators and admins on the various platforms
+  - coordinate efforts for training and processes around moderation
+  - provide support to improve the platforms to lighten moderation and admin workloads
+  - provide training and support to the wider community on how to use the platforms effectively
+
+- What actions are you proposing the WG be allowed to take directly?
+  - Creation and management of processes related to community platforms.
+    - Adding/removing moderators & other roles
+    - Training for moderators
+    - Ensuring moderation is consistent across teams and platforms
```

Comment:
I don't think this can be ensured, but we should still strive.

One issue I think is that Discord skews quite gen Z and especially quite "online" people. I think we tolerate (and probably should, with some guidance for them) things there that probably wouldn't in other places. Personally I find this to be a strength but it has been one of the constant problems. There is a constant tension between people wanting me to ban people and when I do ban people, people telling me I'm doing a bad job for banning them. Which is weird because I only ever banned one person for a CoC violation in the first place. You can probably tell this is a bit of a sore spot so I do think _something_ should be done here.

Reply:
I think this starts with producing actionable guidance and some transparency on how we go about about moderation. This guidance does need to take into account the nuances of each platform and what is natural about each of them.

Reply:
I think I mostly agree that sensitivity to context is important, and that there are behaviors that would be fine in one place and not in another. Like, a common communication pattern in real-time chat platforms is to 

make a longer point
spread out over several messages
one at a time
kinda thinking it through as you go
making a lot of short posts.

That's, like, completely fine in Discord or Slack or whatever, but if someone were to do that on the forum, consistently, or here in Github, we'd probably find it pretty annoying and at some point someone probablly should ask them to conform a bit more to the norms of the platform. 

But on the other hand, thiis needs to have some limits. Context _does_ mater, but only to a point. There are certainly corners of the Discord community where, for example, homophobic slurs are common. That's pretty clearly not OK in official Django spaces.

I was super happy to see this proposal because it seems to be like this group could be a place where these lines can get hashed out. Starting with some core agreed-upon principles that then can be massaged for different contexts seems super cool to me.

Reply:
Yes, completely agreed (although typing
like this
also annoys me on
Discord and Slack
) :)